lib/graphql/relay/relation_connection.rb

```diff
@@ -156,6 +156,14 @@ def paged_nodes_array
         return @paged_nodes_array if defined?(@paged_nodes_array)
         @paged_nodes_array = paged_nodes.to_a
       end
+
+      def paged_nodes_length
+        if paged_nodes.respond_to?(:length)
```

Comment:
AFAIK in ActiveRecord the Relation can be simply checked for length, but in Sequel the query must first be run with `.all` and converted to an array - so I'm reusing `pages_nodes_array` here for that use case.

Reply:
is there anyway to avoid running with .all in sequel.  in my biz, lt is becoming a bottleneck now.